tests/api/api/test_vault.py

```diff
@@ -0,0 +1,48 @@
+from http import HTTPStatus
+
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+from os import environ
+
+import pytest
+from mlrun import mlconf
+from random import randrange
+from mlrun.api import schemas
+
+# Uncomment and set proper values for Vault test (at least one is required)
```

Comment:
Leaving the test for now, let's discuss think about it after this PR is in and the feature will get some mileage
Please rename the file to  `test_secrets.py`